_test_image.sh

```diff
@@ -5,9 +5,17 @@ tmp_dir=$(mktemp -d)
 trap "rm -r \"$tmp_dir\"" EXIT
 echo "hello world" > "$tmp_dir"/test.txt
 for i in {1..100};do
-  echo "hello world" >> "$tmp_dir"/test.txt
```

Comment:
This was important because it made the file large enough to not be inline for the INode. Please add back making test.txt larger than can fit in the INode.

Reply:
done